dev.py

```diff
@@ -426,17 +427,63 @@ def install_project(show_build_log):
             raise
     elapsed = datetime.datetime.now() - start_time
 
-    if ret == 0:
-        print("Installation OK")
-    else:
+    if ret != 0:
         if not show_build_log:
             with open(log_filename, 'r') as f:
                 print(f.read())
         print("Installation failed! ({0} elapsed)".format(elapsed))
         sys.exit(1)
+
+    # On Windows we also need openblas lib installed
+    if platform.system() == 'Windows':
+        ret = copy_openblas()
+    if ret != 0:
+        print("OpenBLAS copy failed!")
+        sys.exit(1)
+    print("Installation OK")
     return
 
 
+def import_module(mod_name, mod_path):
+    # https://stackoverflow.com/a/67692
```

Comment:
We cannot include SO answers verbatim for licensing reasons cf https://scipy.github.io/devdocs/dev/core-dev/index.html#licensing

Reply:
Can that really apply to this particlar tiny recipe?    If so, it would mean that any use of these standard set of calls would trigger a license violation.  Applied this strictly, use of significant parts of standard Python would not longer be compliant with the BSD.

Reply:
We had lengthy discussions about this and I was asked to remove a 6 lines snippets in `_qmc.py` (I actually contacted the original author who granted explicit permission and re-licensed). I know it's (very) annoying especially when it's from a somewhat permissive license... Depending on who you ask, the answer is different and it depends on the country as well. I am not a lawyer and AFAIK we did not get advice from any. If that's the only way to code the logic, it should be fine though. But then I would remove any mention of SO.

Reply:
Ok - will point instead to the Python issue from which this snippet came : https://bugs.python.org/msg255901

Reply:
Thank you 🙇

Reply:
De nada :)